src/lib/market-service/portals/portals.ts

```diff
@@ -138,7 +137,7 @@ export class PortalsMarketService implements MarketService {
             })
 
             marketCapResult[assetId] = {
-              price: bnOrZero(token.pricePerShare).toFixed(),
+              price: bnOrZero(token.price).toFixed(),
```

Comment:
We got it just as wrong as Zapper (not that it would work either way, since Coingecko was taking priority for LP-YCRV the issue was spotted for, and probably other Yearn LP tokens too)